.github/workflows/link-check.yml

```
@@ -0,0 +1,17 @@
+name: Link Check
+on:
+  push:
+  pull_request:
+  schedule:
+    # Run every day at 9:00 AM (See https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07)
+    - cron: "0 9 * * *"
+jobs:
+  link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

i would pin the action
```suggestion
      - uses: actions/checkout@v3
```

---

fixed